src/sage/topology/simplicial_set_constructions.py

```diff
@@ -170,7 +171,7 @@ def __init__(self, data, ambient=None):
             ambient = self
         if (ambient.is_pointed()
             and hasattr(ambient, '_basepoint')
-            and ambient.base_point() in data):
+                and ambient.base_point() in data):
```

Comment:
indent previous line?

Reply:
this is the way my editor pycodestyle-plugin like the things to be written. Then it displays no warning. But this also works fine if I indent the line above, right.